index.html

```diff
@@ -34,6 +34,35 @@
         <a href="#about"><p>About</p></a>
         <a href="#contact"><p>Contact</p></a>
       </nav>
+    </div>
+    <div id="overlay">s
+      <div id="popup">
+        <div class="popup-header">
+          <h4 id="popup-project-heading" class="popup-heading">This Heading</h4>
+          <i id="close-btn" href="#" onclick="PopupWindowOff()" class="fa-solid fa-close fa-2x close-popup"></i>
+        </div>
+        <div class="popup-body">
+          <div class="popup-image-area">
+            <img id="popup-project-image" alt="" />
+          </div>
```

Comment:
- [ ] The image should stretch all the width of the popup: 

Your design  | Figma design
------------- | -------------
<img width="873" alt="Screenshot 2022-09-09 at 15 47 15" src="https://user-images.githubusercontent.com/77038610/189353230-0dd06b29-a1c7-4370-932c-212a7dd2241e.png"> | <img width="557" alt="Screenshot 2022-09-09 at 15 47 46" src="https://user-images.githubusercontent.com/77038610/189353311-ed93f8d4-3538-4667-a9b2-25b0f0036974.png">

Hint, you can use `object-fit: cover;` property in CSS to achieve that  😉